unity-environment/README.md

```diff
@@ -43,10 +43,10 @@ For information on how to create a new Unity Environment, see the walkthrough [h
 If you will be using Tensorflow Sharp in Unity, you must:
 
 1. Make sure you are using Unity 2017.1 or newer.
-2. Make sure the TensorflowSharp plugin is in your Asset folder. A Plugins folder which includes TF# can be downloaded [here](https://s3.amazonaws.com/unity-agents/TFSharpPlugin.unitypackage).
+2. Make sure the TensorflowSharp [plugin](https://s3.amazonaws.com/unity-agents/TFSharpPlugin.unitypackage) is in your Asset folder.
```

Comment:
Shouldn't the link be https://s3.amazonaws.com/unity-agents/0.2/TFSharpPlugin.unitypackage ?

Reply:
good catch. Updating now